crates/rolldown/src/css/mod.rs

```diff
@@ -28,10 +30,25 @@ pub fn create_css_view(
           SymbolRef::from((ModuleIdx::from_raw(0), SymbolId::from_usize(0))),
           range.start,
         ));
+        let mut range_end = range.end as usize;
+        if source.is_char_boundary(range_end) {
+          let next_char = source[range_end..].chars().next();
```

Comment:
`starts_with` should be better, since `\n` is just a ascii char

Reply:
Done